src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -24,41 +22,6 @@ namespace Microsoft.Kusto.ServiceLayer.DataSource.DataSourceIntellisense
     /// </summary>
     public class KustoIntellisenseHelper
     {
```

Comment:
Moved the 3 classes into separate files

Reply:
You could do this in the next checkin if it is easier, but move all the Kusto specific files into a "Kusto" directory under the DataSource directory. Hope to add directories for other data sources soon.